Mathlib/RingTheory/OreLocalization/Basic.lean

```diff
@@ -84,91 +87,94 @@ variable (R S) [OreSet S]
 
 @[inherit_doc OreLocalization]
 scoped syntax:1075 term noWs atomic("[" term "⁻¹" noWs "]") : term
-macro_rules | `($R[$S⁻¹]) => ``(OreLocalization $R $S)
+macro_rules | `($R[$S⁻¹]) => ``(OreLocalization $S $R)
 
 attribute [local instance] oreEqv
 
 variable {R S}
+variable {X} [MulAction R X]
 
-/-- The division in the ore localization `R[S⁻¹]`, as a fraction of an element of `R` and `S`. -/
-def oreDiv (r : R) (s : S) : R[S⁻¹] :=
+/-- The division in the ore localization `X[S⁻¹]`, as a fraction of an element of `X` and `S`. -/
```

Comment:
Can we make the notation `X[S⁻¹]` work?

It also seems that many docstrings below need some updating.

Reply:
Turns out the notation already works. I just thought it didn't.